engine/api/api.go

```diff
@@ -45,8 +45,9 @@ type Configuration struct {
 		UI  string `toml:"ui" default:"http://localhost:2015"`
 	} `toml:"url" comment:"#####################\n CDS URLs Settings \n####################"`
 	HTTP struct {
-		Port       int `toml:"port" default:"8081"`
-		SessionTTL int `toml:"sessionTTL" default:"60"`
+		Addr       string `toml:"addr" default:"127.0.0.1" commented:"true" comment:"Listen address without port"`
```

Comment:
humm not sure it's good idea to set by default 127.0.0.1, i won't be enough with the docker-compose file; 0.0.0.0 (= all interfaces) should work